extensions/linz/README.md

```diff
@@ -75,6 +75,7 @@ See [ISO/IEC 13249-3:2016(en)](https://www.iso.org/obp/ui/#!iso:std:60343:en) fo
 | linz:providers                 | LINZ Provider Object | **REQUIRED**. The object provides information about a provider with additional roles defined by Toitū Te Whenua LINZ. A provider is any of the organizations that captures or processes the content of the assets and therefore influences the data offered by the STAC implementation. See [LINZ Provider Object](#linz-provider-object).       |
 | providers                      | Provider Object      | **REQUIRED**. The object provides information about a provider. A provider is any of the organizations that captures or processes the content of the assets and therefore influences the data offered by the STAC implementation. See [Provider Object](#provider-object).                                                                       |
 | linz:security_classification   | string               | **REQUIRED**. New Zealand Government [Security Classification](https://www.digital.govt.nz/standards-and-guidance/governance/managing-online-channels/security-and-privacy-for-websites/foundations/classify-information/). Must be one of `unclassified`, `in-confidence`, `sensitive`, `restricted`, `confidential`, `secret` or `top-secret`. |
+| linz:update_frequency          | string               | Recommended. Indicates how frequently an updated dataset may be distributed to the publication platform. Must follow the format for durations as defined in [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339#appendix-A). For example, `P3D` expresses a duration of 3 days.                                                              |
```

Comment:
"T" is only if you specify hours, minutes, or seconds, as far as I know. Pretty sure `P3D` is allowed. [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339#appendix-A) says

> ISO 8601 states that the "T" may be omitted under some   circumstances.  This grammar requires the "T" to avoid ambiguity.

I think that applies only in case of datetimes, not durations. I'll try to work out the grammar…

Reply:
According to [Wikipedia](https://en.wikipedia.org/wiki/ISO_8601#Durations):

> Date and time elements including their designator may be omitted if their value is zero, and lower-order elements may also be omitted for reduced precision. For example, "P23DT23H" and "P4Y" are both acceptable duration representations. However, at least one element must be present, thus "P" is not a valid representation for a duration of 0 seconds. "PT0S" or "P0D", however, are both valid and represent the same duration.

Reply:
The RFC 3999 [ABNF](https://en.wikipedia.org/wiki/Augmented_Backus%E2%80%93Naur_form) definition for durations:

```
Durations:

   dur-second        = 1*DIGIT "S"
   dur-minute        = 1*DIGIT "M" [dur-second]
   dur-hour          = 1*DIGIT "H" [dur-minute]
   dur-time          = "T" (dur-hour / dur-minute / dur-second)
   dur-day           = 1*DIGIT "D"
   dur-week          = 1*DIGIT "W"
   dur-month         = 1*DIGIT "M" [dur-day]
   dur-year          = 1*DIGIT "Y" [dur-month]
   dur-date          = (dur-day / dur-month / dur-year) [dur-time]

   duration          = "P" (dur-date / dur-time / dur-week)
```

`/` means "xor" and `[…]` means *optional*, so a date-only duration (`dur-date`) is *optionally* followed by a time duration. Since the "T" is part of the time duration, it's only needed if some time part is specified.

For example, `P1M` means one month, while `PT1M` means one minute.

Reply:
Okay. Is there a better link for this standard then?

Reply:
What do you mean by "better"? Do you need stronger proof this is how it works?

Reply:
No, just some clearer explanation of how it works. But if reading the 'durations' table in that link makes sense to most people it's probably okay. Pluse Mitch added a simple example `P3D`, so that's probably enough.